setup.cfg

```diff
@@ -11,3 +11,7 @@ versionfile_build = flocker/_version.py
 tag_prefix =
 parentdir_prefix = Flocker-
 
+[flake8]
+# Ignore the following annoying pep8 errors:
+# E402: module level import not at top of file
+ignore = E402
```

Comment:
I think this triggers a break in the lint builder: http://ci-live.clusterhq.com:8080/job/ClusterHQ-flocker/job/list-live-nodes-3467/job/run_lint/2/console